### PR TITLE
Make DG pdf

### PR DESCRIPTION
**Pull request by EksdeeeX**
- Edit spacing and line break for DG pdf

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -743,16 +756,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Glossary
 
+* **Mainstream OS**: Windows, Linux, Unix, macOS
+* **Command Line Interface (CLI)**: A text-based user interface (UI) used to run programs, manage computer files and interact with the computer
+* **User Interface (UI)**: A form of user interface that allows users to interact with electronic devices through graphical icons
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
```
Comment (glozxi) [inline docs/DeveloperGuide.md:762]: Should this line be deleted?

**Review by glozxi — CHANGES_REQUESTED** [review]
LGTM, just a nit

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (EksdeeeX) [reply docs/DeveloperGuide.md:762]: resolved!

**Review by glozxi — APPROVED** [review]
LGTM

**Pull request merged**